docs/_components/modals-basic.html

```diff
@@ -0,0 +1,25 @@
+---
+title: "Basic"
+description: |
```

Comment:
@jmatembu 

>I think you got some time to get the new changes in as we receive components from other contributors.

I'll work on them soon will definitely get back to you today.

>Did you manage to see the showcase website locally?

Yes. I ran the command `npm run dev-docs` in another terminal.